internal/sdk/cloudian/sdk.go

```diff
@@ -108,6 +108,12 @@ func toInternalUser(u User) userInternal {
 	}
 }
 
+// The cloudian API doc names this data `SecurityInfo` so we retain that name.
```

Comment:
```suggestion
// SecurityInfo is the Cloudian API's term for secure credentials
```

As with the func, start with the type name (or _A TypeName_). Also prefer comment to be a bit more formal, and not state the obvious that _we retain that name_.

https://tip.golang.org/doc/comment#type